Neos.Fusion/Classes/FusionObjects/Helpers/FusionPathProxy.php

```diff
@@ -140,11 +141,12 @@ public function offsetUnset($offset)
      * Evaluates Fusion objects and eel expressions.
      *
      * @return FusionPathProxy|mixed
+     * @throws FusionException
      */
     public function objectAccess()
     {
         if (!$this->fusionRuntime->canRender($this->path)) {
-            return $this;
+            throw new FusionException('The configuration in the path "' . $this->path . '" can not be rendered.', time());
```

Comment:
This timestamp here is regenerated every time and is therefore not suitable for an exception identifier. The timstamp needs to be generated separately and added als fix number here. No worries, you are not the first one stumbled over this ;)